src/notifications/notifications.module.ts

```
@@ -1,8 +1,14 @@
-import { Module } from '@nestjs/common';
+import { forwardRef, Module } from '@nestjs/common';
+import { NotificationsService } from './notifications.service';
+import { NotificationsController } from './notifications.controller';
+import { NotificationSettingsModule } from '../notification-settings/notification-settings.module';
 import { TypeOrmModule } from '@nestjs/typeorm';
+import { JobNotification } from './entities/job-notification.entities';
+import { PolicyVersionModule } from '../policy-version/policy-version.module';
+import { ConnectionNotification } from './entities/connection-notification.entity';
+
+import { Module } from '@nestjs/common';
```

Biome (1.9.4)

[error] Shouldn't redeclare 'Module'. Consider to delete it or rename it.

'Module' is defined here:


(lint/suspicious/noRedeclare)